tests/Application/app/config/routing.yml

```diff
@@ -1,8 +1,12 @@
 sylius:
     resource: "../../../../vendor/sylius/sylius/app/config/routing.yml"
 
-# Put your own routes here
-
-sylius_invoicing_plugin:
-    resource: "@SyliusInvoicingPlugin/Resources/config/app/routing.yml"
+sylius_invoicing_plugin_admin:
+    resource: "@SyliusInvoicingPlugin/Resources/config/app/routing/admin_invoicing.yml"
```

Comment:
Let's follow the convention we use for core bundles and PluginSkeleton and put routing in `@SyliusInvoicingPlugin/Resources/config/admin_routing.yml` and `@SyliusInvoicingPlugin/Resources/config/shop_routing.yml` respectively.